pages/content/posts/2022-q3-report/index.md

```diff
@@ -0,0 +1,178 @@
+---
+author: "Open Web Docs staff"
+title: "Open Web Docs 2022 Q3 report"
+date: "2022-10-27"
+ShowToc: true
+ShowBreadCrumbs: false
+excerpt: "An update on our work in the third quarter of 2022."
+---
+
+<div class="callout">
+
+In 2022 Q3, OWD:
+
+- was again the main organization maintaining the [mdn/content](https://github.com/mdn/content) repository:
```

Comment:
Is there a specific reason all of these bullets start with lowercase?

Reply:
```suggestion
- was again the main organization maintaining the [mdn/content](https://github.com/mdn/content) repository:
```

- was the major contributor to the [mdn/content](https://github.com/mdn/content) repository:

Reply:
It starts with lowercase because it's not a complete sentence. But I'm not prescriptive about things like this and think it is a subjective style guide thing, so don't mind changing if you think it is better.

Reply:
Yeah, I don't feel too strongly either, was mostly curious. :-)